DataConnectors/Qualys VM/AzureFunctionQualysVM/run.ps1

```diff
@@ -201,7 +200,6 @@ if (-not ($response.HOST_LIST_VM_DETECTION_OUTPUT.RESPONSE.HOST_LIST -eq $null))
         # Convert the array containing all the records to JSON and API POST to Log Analytics Workspace                                                                 
         $json = $newcustomObjects | ConvertTo-Json -Compress -Depth 3                                                            
         Post-LogAnalyticsData -customerId $customerId -sharedKey $sharedKey -body ([System.Text.Encoding]::UTF8.GetBytes($json)) -logType $TableName
-        Write-Output $json
```

Comment:
Wouldn't removing this cause the json output to not be sent to the Log Analytics data collection API? Or is this a logging statement?

Reply:
the json object is posted into log analytics, this is writing out the output in the logs /console and is unnecessary